src/Event.php

```diff
@@ -195,6 +200,11 @@ protected function setDateProperty(string $name, Carbon $date)
         }
     }
 
+    public function addAttendees(array $attendees)
+    {
+        $this->attendees[] = $attendees;
```

Comment:
This looks off. Shouldn't it be either:

```php
$this->attendees = array_merge($this->attendees, $attendees);
```

or

```php
$this->attendees = $attendees;
```

(I prefer the former)

As it is now, you'll end up with an array of arrays.